podpac/core/node.py

```diff
@@ -74,8 +77,14 @@ class Node(tl.HasTraits):
 
     Attributes
     ----------
+    cache_output: bool
+        Default if True. Should the node's output be cached? 
+    cache_update: bool
+        Default is False. Should the node's cached output be updated from the source data? 
     cache_type : [None, 'disk', 'ram']
```

Comment:
We'll have to think a little more about this attribute... what if I want ram and disk?

Reply:
Agreed, `CacheCtrl` should probably be able to accept a collection of cache mode strings (list/set), then this could be a collection too.